src/utils/ltfs_ordered_copy

```diff
@@ -60,14 +66,14 @@ class CopyItem:
 
     def eval(self): #access to the extended attributes present in some operating systems/filesystems by xattr
         try:
-            self.vuuid = xattr.get(self.src, self.vea_pre + 'ltfs.volumeUUID')
+            self.vuuid = xattr.getxattr(self.src, self.vea_pre + 'ltfs.volumeUUID')
         except Exception as e:
             self.vuuid = ''
             return (self.vuuid, self.part, self.start)
 
         try:
-            self.part  = xattr.get(self.src, self.vea_pre + 'ltfs.partition')
-            start_str  = xattr.get(self.src, self.vea_pre + 'ltfs.startblock')
+            self.part  = xattr.getxattr(self.src, self.vea_pre + 'ltfs.partition')
```

Comment:
Are you using the [`pyxattr`](https://pypi.org/project/pyxattr/) module? Now we are expecting the [`xattr`](https://pypi.org/project/xattr/) module. And I think `xattr.getxattr` is not provided in the `xattr` package.

The reason why we assume the `xattr` module is it is provided by the `pyhon-xattr` on the official repository of RHEL7/8. As you see, the first target of this project is RHEL7 and RHEL8.

So I cannot accept this change. Please create another `ltfs_ordered_copy` for `pyxattr`.

Reply:
In FreeBSD I use [xattr](https://github.com/xattr/xattr), which provides xattr.getxattr and alike.

I don't have access to RHEL7/8 but I've seen some documents from redhat with a list of packages and their licenses, an I see one called pyxattr (in RHEL7) and one called python3-pyxattr (in RHEL8) with LGPLv2+ license. Based on that, I assume that RHEL is really using [pyxattr](https://github.com/iustin/pyxattr) module (because the other xattr is MIT licensed).

The pyxattr module (the one I assume is in RHEL) deprectared xattr.getxattr some time ago in favor of xattr.get. But the deprecated functions are still available (at least this is what I see in the main branch of pyxattr in github).

So, by now the use of xattr.xattrget works on both modules.

I understand you may have concerns with this PR because we don't know what the pyxattr folks will do regarding these deprecated functions. So, maybe the best way to deal with that is to only merge the errno handling code, and then I may ask the FreeBSD port maintainer to do the rest of the patch there.

Reply:
Oops, sorry I was completely confused and I need to apologize to you. 

The module we are using is `pyxattr`, so we can accept this. Let me have more time  to confirm that this modification doesn't break the behavior on RHEL7/8.

The [IBM docs](https://www.ibm.com/docs/en/spectrum-archive-sde/2.4.5?topic=planning-required-software) says it is `pyxattr`.

Reply:
But I think it is a good idea to sepalate `errno` fix and `getxattr` fix. I can merge the `errno` fix ASAP as you mentioned.

Reply:
Oops, sorry for keeping as is so long time.

I decided to merge this on the `master` branch. We have a time to test it before releasing from this branch!